test/Utilities/Microsoft.Testing.TestInfrastructure/Microsoft.Testing.TestInfrastructure.csproj

```diff
@@ -7,18 +7,17 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Testing.Internal.Framework" />
+    <PackageReference Include="MSTest.TestFramework" />
```

Comment:
Should we explicitly reference MSTest.Analyzers here as well? At least until we are using 3.8 which will have https://github.com/microsoft/testfx/pull/4351

Reply:
Yes!